cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -115,3 +116,658 @@ def test_validate_not_required_dialog_element():
         1692736
     """
     pass
+
+
+@pytest.mark.meta(coverage=[1696474])
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_dynamic_dialog_fields_ansible_tower_templates():
+    """
+
+    Bugzilla:
+        1696474
+
+    Polarion:
+        assignee: nansari
+        startsin: 5.10
+        casecomponent: Services
+        initialEstimate: 1/2h
+        testSteps:
+            1. Add dynamic dialog with "text area" field
+            2. Add service catalog with above created dialog
+            3. Order the service
+        expectedResults:
+            1.
+            2.
+            3. Populated "text area" fields should pass correct value to ansible tower templates
+    """
+    pass
+
+
+@pytest.mark.meta(coverage=[1707961])
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_dialog_editor_modify_field():
+    """
+
+    Bugzilla:
+        1707961
+
+    Polarion:
+        assignee: nansari
+        startsin: 5.10
+        casecomponent: Services
+        initialEstimate: 1/16h
+        testSteps:
+            1. Add dialog
+            2. Edit a dialog
+            3. Edit and save as many fields as many times as you like
+            4. Exit without saving the dialog
+        expectedResults:
+            1.
+            2.
+            3.
+            4. The UI should confirm that you want to exit without saving your dialog
+    """
+    pass
+
+
+@pytest.mark.meta(coverage=[1706848])
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_specific_dates_and_time_in_timepicker():
+    """
+
+    Bugzilla:
+        1706848
+
+    Polarion:
+        assignee: nansari
+        startsin: 5.10
+        casecomponent: Services
+        initialEstimate: 1/16h
+        testSteps:
+            1. Create a dialog with timepicker
+            2. Select specific dates and time, Save
+        expectedResults:
+            1.
+            2. Able to set specific dates and time in timepicker
+    """
+    pass
+
+
+@pytest.mark.meta(coverage=[1706693])
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_dynamic_field_on_refresh_button():
+    """
+
+    Bugzilla:
+        1706693
+
+    Polarion:
+        assignee: nansari
+        startsin: 5.10
+        casecomponent: Services
+        initialEstimate: 1/16h
+        testSteps:
+            1. Import Datastore and dialog
+            2. Add service catalog with above created dialog
+            3. In Order page of service
+        expectedResults:
+            1.
+            2.
+            3. dynamic field shouldn't be blank
+    """
+    pass
+
+
+@pytest.mark.meta(coverage=[1705021])
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_refresh_of_dynamic_field_contains_word_password():
```

Comment:
Can this test case be merged into the previous test case? They seem to be testing the same page.

Reply:
Here in service dialog lots of tests testing on the same page but it has different different scenario and automate method so we can't merge it.